code/go/0chain.net/blobbercore/allocation/protocol.go

```diff
@@ -50,6 +50,12 @@ func (a *Allocation) LoadTerms(ctx context.Context) (err error) {
 
 // VerifyAllocationTransaction try to get allocation from postgres.if it doesn't exists, get it from sharders, and insert it into postgres.
 func VerifyAllocationTransaction(ctx context.Context, allocationTx string, readonly bool) (a *Allocation, err error) {
+	t, err := transaction.VerifyTransaction(allocationTx, chain.GetServerChain())
```

Comment:
Actually Previous logic is better than this because if the allocation transaction `allocationTx`( which is different from `allocationID`) is same as it is in the database then we don't need to verify the transaction.

Transaction verification is costlier than retrieving from database. So Please revert it back.

Note; `allocationID` is the `allocationTx` in initial allocation creation time. Later when allocation is updated then these two differ.

Reply:
Okay, will revert.